reference/transactions/overview.md

```diff
@@ -209,3 +209,5 @@ COMMIT;
 为了使性能达到最优，建议每 100～500 行写入一个事务。
 
 TiDB 设置了单个事务的键值对的总大小不超过 100 MB，这个默认值可以通过配置文件中的配置项 `txn-total-size-limit` 进行修改，最大支持到 10GB。实际的单个事务大小限制还取决于用户的内存，执行大事务时 TiDB 进程的内存消耗大约是事务大小 6 倍以上。
+
+在 4.0 以前的版本，TiDB 限制了单个事务的键值对的总数量不超过 30 万条，在 4.0 之后的版本没有这项限制。
```

Comment:
Please update this part in [#2002](https://github.com/pingcap/docs/pull/2002).

Reply:
There is nothing about it here https://github.com/pingcap/docs/blob/3a61188f3c8cd78369d50f2553edcbbd3273b022/reference/transactions/overview.md

Reply:
The whole block is missing

Reply:
This is not the latest document. This is : https://github.com/pingcap/docs/blob/master/reference/transactions/overview.md.

Reply:
@tiancaiamao The missing part is aligned in https://github.com/pingcap/docs/pull/2280. PTAL